sdk/search/azure-search/src/main/java/com/azure/search/models/SuggestOptions.java

```diff
@@ -0,0 +1,325 @@
+     * disabled.
+     */
+    @JsonProperty(value = "")
+    private String highlightPreTag;
+
+    /*
+     * A number between 0 and 100 indicating the percentage of the index that
+     * must be covered by a suggestions query in order for the query to be
+     * reported as a success. This parameter can be useful for ensuring search
+     * availability even for services with only one replica. The default is 80.
+     */
+    @JsonProperty(value = "")
+    private Double minimumCoverage;
+
+    /*
+     * The list of OData $orderby expressions by which to sort the results.
+     * Each expression can be either a field name or a call to either the
+     * geo.distance() or the search.score() functions. Each expression can be
+     * followed by asc to indicate ascending, or desc to indicate descending.
+     * The default is ascending order. Ties will be broken by the match scores
+     * of documents. If no $orderby is specified, the default sort order is
+     * descending by document match score. There can be at most 32 $orderby
+     * clauses.
+     */
+    @JsonProperty(value = "")
+    private List<String> orderBy;
+
+    /*
+     * The list of field names to search for the specified search text. Target
+     * fields must be included in the specified suggester.
+     */
+    @JsonProperty(value = "")
+    private List<String> searchFields;
+
+    /*
+     * The list of fields to retrieve. If unspecified, only the key field will
+     * be included in the results.
+     */
+    @JsonProperty(value = "")
+    private List<String> select;
+
+    /*
+     * The number of suggestions to retrieve. The value must be a number
+     * between 1 and 100. The default is 5.
+     */
+    @JsonProperty(value = "")
+    private Integer top;
+
+    /**
+     * Get the filter property: An OData expression that filters the documents
+     * considered for suggestions.
+     *
+     * @return the filter value.
+     */
+    public String getFilter() {
+        return this.filter;
+    }
+
+    /**
+     * Set the filter property: An OData expression that filters the documents
+     * considered for suggestions.
+     *
+     * @param filter the filter value to set.
+     * @return the SuggestOptions object itself.
+     */
+    public SuggestOptions setFilter(String filter) {
+        this.filter = filter;
+        return this;
+    }
+
+    /**
+     * Get the useFuzzyMatching property: A value indicating whether to use
+     * fuzzy matching for the suggestions query. Default is false. When set to
+     * true, the query will find terms even if there's a substituted or missing
+     * character in the search text. While this provides a better experience in
+     * some scenarios, it comes at a performance cost as fuzzy suggestions
+     * queries are slower and consume more resources.
+     *
+     * @return the useFuzzyMatching value.
+     */
+    public Boolean isUseFuzzyMatching() {
```

Comment:
Same issue as `MagnitudeScoringParameters.isShouldBoostBeyondRangeByConstant`

Reply:
since useFuzzyMatching is boolean, autorest names the getter with "is"

Reply:
Yes, I would argue that it's a bug. We'll look into it after this PR is merged.